assets/section-image-banner.css

```diff
@@ -2,10 +2,13 @@
   display: flex;
```

Comment:
In the default state there isnt space between the two buttons: https://screenshot.click/24-46-wkm79-tduo9.png

Reply:
My bad. I missed the `display: flex` on the container. Just updated that.